src/extensions/surveys.tsx

```diff
@@ -72,10 +72,7 @@ export const callSurveys = (posthog: PostHog, forceReload: boolean = false) => {
                     }
                 }
             }
-            if (
-                survey.type === SurveyType.Popover &&
-                document.querySelectorAll("div[class^='PostHogSurvey']").length === 0
-            ) {
+            if (survey.type === SurveyType.Popover) {
```

Comment:
hmm, is it safe to remove that query selector? I think it prevents showing a 2nd survey while one survey is already being shown. Since they are different surveys, I don't think we have any other mechanism to prevent this

Reply:
yeah, I moved the `seenSurvey` check here to prevent it, I think I might have to do something else here. since the query selector prevents any new surveys without refreshing the page.

Reply:
fixed now